### PR TITLE
🚚 Mass rename of CavaticaTask and relationships

### LINKED ISSUE #465: Generalize/Rename CavaticaTask Models
1. Rename/refactor CavaticaTask, CavaticaTaskGenomicFile models to be more generic. Something like TaskModel, or BioinformaticsTaskModel, etc
2. <Fill in> @allisonheath

### PR DESCRIPTION
**Pull request by dankolbman**
Changes `CavaticaTask` and `CavaticaTaskGenomicFile` to more general `Task` and `TaskGenomicFile` fixes #465 .

### CODE REVIEW

File: dataservice/api/task/models.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+    :param kf_id: Unique id given by the Kid's First DCC
+    :param created_at: Time of object creation
+    :param modified_at: Last time of object modification
+    :param name: Name of the task
+    :param external_task_id: Id of executed task assigned by the external
+        informatics tool, eg: Cavatica or Firecloud.
+    """
+    __tablename__ = 'task'
+    __prefix__ = 'TB'
```
Comment (znatty22) [inline dataservice/api/task/models.py]: Just curious why prefix isn't TA or TK? What's the B mean?

File: dataservice/api/task/__init__.py [hunk 2]
```diff
@@ -0,0 +1,2 @@
+from dataservice.api.task.resources import TaskAPI
```
Comment (znatty22) [inline dataservice/api/task/__init__.py:1]: Github won't let me put a comment on dataservice/api/task/README.md but i'll put my comment here. README has to be updated:
This
```
ID Prefix: `CT`
```
should be:
```
ID Prefix: <whatever prefix we decide>
```

Comment (znatty22) [reply dataservice/api/task/__init__.py:1]: Same comment for dataservice/api/task_genomic_file/README.md

File: dataservice/api/task/models.py (hunk 1 above)
Comment (dankolbman) [reply dataservice/api/task/models.py]: TA is used by the release coordinator. We can go with TK

**Review by fiendish — APPROVED** [review]

**Comment by fiendish** [comment]
Are those codacy warnings going to be addressed?

File: dataservice/api/task_genomic_file/schemas.py [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+from dataservice.api.task.models import TaskGenomicFile
+from dataservice.api.common.schemas import BaseSchema
+from dataservice.extensions import ma
+
+
+class TaskGenomicFileSchema(BaseSchema):
+
+    task_id = field_for(TaskGenomicFile, 'task_id',
+                        required=True, load_only=True,
+                        example='TB_ABB2C104')
```
Comment (znatty22) [inline dataservice/api/task_genomic_file/schemas.py]: `example='TK_ABB2C104'`

**Comment by dankolbman** [comment]
@fiendish looks like the codacy configs got a little mixed up and need to be re-run. I've been ignoring them lately because they take several hours to come back for some reason.

**Review by znatty22 — APPROVED** [review]
👍

**Pull request merged**